react/features/base/tracks/loadEffects.web.js

```diff
@@ -20,17 +19,9 @@ export default function loadEffects(store: Object): Promise<any> {
             .catch(error => {
                 logger.error('Failed to obtain the background effect instance with error: ', error);
 
-                return Promise.resolve();
-            })
-        : Promise.resolve();
-    const screenshotCapturePromise = state['features/screenshot-capture']?.capturesEnabled
-        ? createScreenshotCaptureEffect(state)
-            .catch(error => {
-                logger.error('Failed to obtain the screenshot capture effect effect instance with error: ', error);
-
                 return Promise.resolve();
             })
         : Promise.resolve();
 
-    return Promise.all([ backgroundPromise, screenshotCapturePromise ]);
+    return Promise.all([ backgroundPromise ]);
```

Comment:
This doesn't need to be an array if there is just one element, just return the promise directly.

Reply:
i thought the same, but it seems that it might be nice for adding effects later, since that's what the function name implies. I guess it's a matter of preference

Reply:
Turns out it's actually needed as the function that calls loadEffects expects the result to be an array